views/tx.tmpl

```diff
@@ -99,7 +99,7 @@
                                         >
                                             <span class="nowrap pl-1 pr-1">
                                               {{if gt .Confirmations 0}}
-                                                Immature, {{ if eq .Type "Ticket"}}eligible to vote{{else}}spendable{{end}} in {{ if eq .Confirmations .TicketInfo.TicketMaturity }}next block{{else}}{{subtract (add .TicketInfo.TicketMaturity 1) .Confirmations}} blocks ({{printf "%.1f" .TicketInfo.TimeTillMaturity}} hours remaining){{end}}
+                                                Immature, {{ if eq .Type "Ticket"}}eligible to vote{{else}}spendable{{end}} in {{ if eq .Confirmations .TicketInfo.TicketMaturity }}next block{{else}}{{subtract .TicketInfo.TicketMaturity .Confirmations}} blocks ({{printf "%.1f" .TicketInfo.TimeTillMaturity}} hours remaining){{end}}
```

Comment:
I had to check with others to be sure, but I believe a ticket is not considered mature until 257 confirmations (256 blocks after it is mined).

Reply:
@davecgh I saw you posted something to this effect in Slack, but I'd like if you would confirm the above.

Reply:
So the `TicketMaturity` comes from `chaincfg.Params` and is `256`. With this new information about 257 needed confirmation (inc. first block), I can now see why it was that way. Just to make things more confusing, the `aria-valuemax` of the progress bar is set to `256`, where I guess it should be 257 as well.

Reply:
mm hmm, please see https://decred.slack.com/archives/C3ZNV8QTX/p1540938352047200 (Dave's response).
I think we've been inaccurate in some frontend spots regarding maturity.

Reply:
Yes, the maturity is 256, but stake transactions don't affect the current block in terms of things like the ticket pool.  So, it does not actually become eligible to vote until the 257th block.  In other words, it matures in the 256th block and gets added to the pool _after_ the winning tickets and such have been selected, ergo it isn't actually eligible to be selected until the 257th.

Reply:
Alright, so we have to decide if the progress bar should maturity, or blocks until eligibility to vote.  :laughing:  Because it sounds like it would be correct to have the last step in the progress bar be "Mature, eligible to vote in next block" with 100% progress, where the previous block is "Immature, eligible to vote in two blocks" with 99.61% progress (255/256).

Reply:
I think that would be the best approach. Show maturity and clarify eligibility.

Reply:
@buck54321 Does that suggestion make sense, and would it be straightforward to implement?

Reply:
Yeah. I'm kinda in a fog of `+1`'s and `-1`'s, but I think I get it.